Arrowgene.Ddon.Database/Files/Database/Script/migration_quest_variant.sql

```diff
@@ -0,0 +1,13 @@
+DELETE FROM "ddon_quest_progress" WHERE "quest_id" = 2005501;
+
+DELETE FROM "ddon_priority_quests" WHERE "quest_id" = 2005501;
+
+DELETE FROM "ddon_reward_box" WHERE "quest_id" = 2005501;
+
+DELETE FROM "ddon_completed_quests" WHERE "quest_id" = 2005501;
```

Comment:
Wouldnt it be better to update these entries so they point to the new ID insteaad of outright dropping it?

Reply:
Changed this now to update tables with the correct info :)